httpcache_test.go

```diff
@@ -885,6 +885,9 @@ func TestUpdateFields(t *testing.T) {
 }
 
 func TestCachedErrorsKeepStatus(t *testing.T) {
+	// tests fix for https://github.com/gregjones/httpcache/issues/74
+	// previously, after validating a cached response, its StatusCode
+	// was incorrectly being replaced
```

Comment:
Minor, but I'd suggest moving the comment out to be on top of the entire test. It describes what the entire test does. E.g., compare with:

https://github.com/gregjones/httpcache/blob/cdafe69ab9c9f0346f6345a6c475e0c170f35dbb/httpcache_test.go#L173-L175

https://github.com/gregjones/httpcache/blob/cdafe69ab9c9f0346f6345a6c475e0c170f35dbb/httpcache_test.go#L1449-L1454

Also, I'd suggest using proper sentences that start with capital letters with periods at end. It makes it harder to read the text without periods. At first, I thought it was one sentence and felt confused. After, I realized that there are 2 sentences here, and the second sentence begins with "Previously, ...".